modules/graceful/server.go

```diff
@@ -128,6 +128,8 @@ func (srv *Server) ListenAndServeTLS(certFile, keyFile string, serve ServeFuncti
 func (srv *Server) ListenAndServeTLSConfig(tlsConfig *tls.Config, serve ServeFunction) error {
 	go srv.awaitShutdown()
 
+	tlsConfig.MinVersion = tls.VersionTLS12
```

Comment:
@kaepora can you remove this from **this** pull and resolve confilct
then I think this is ready to mrege

and pleace make a second pull for **tlsConfig.MinVersion = tls.VersionTLS12**